File: lib/models/stylist.dart
class Stylist {
  final String id;
  final String salonId;
  final String name;
  final String level; // "Thợ chính" | "Thợ phụ"
  final double rating;

  const Stylist({
    required this.id,
    required this.salonId,
    required this.name,
    required this.level,
    required this.rating,
  });

  factory Stylist.fromJson(Map<String, dynamic> json) {
    return Stylist(
      id: json['id'] as String,
      salonId: json['salonId'] as String,
      name: json['name'] as String,
      level: json['level'] as String,
      rating: (json['rating'] as num).toDouble(),
    );
  }
}

File: backend/src/models/Stylist.js
const { Schema, model } = require('mongoose');

const stylistSchema = new Schema({
  salonId: { type: Schema.Types.ObjectId, ref: 'Salon', required: true },
  name: { type: String, required: true },
  level: { type: String, default: 'Thợ chính' },
  rating: { type: Number, default: 4.5 },
});

module.exports = model('Stylist', stylistSchema);
